py/torch_tensorrt/dynamo/backend/__init__.py

```diff
@@ -45,6 +49,10 @@ def compile(
     min_block_size=MIN_BLOCK_SIZE,
     torch_executed_ops=[],
     torch_executed_modules=[],
+    max_aux_streams=MAX_AUX_STREAMS,
+    version_compatible=VERSION_COMPATIBLE,
+    optimization_level=OPTIMIZATION_LEVEL,
+    use_experimental_rt=USE_EXPERIMENTAL_RT,
```

Comment:
If we are adding the experimental runtime, we should move those changes into a new PR which also moves the RT into dynamo and have that PR depend on this

Reply:
This PR has now been refactored into the new PR